packages/apps/tools/src/components/Common/Layout/index.tsx

```diff
@@ -31,13 +50,91 @@ export const Layout: FC<IProps> = ({ children }: IProps) => {
     },
   ];
 
+  const getNetworks = (): Network[] => {
+    const result = new Set(
+      accounts
+        ?.map((account) => {
+          const [, network] = account.split(':');
+          return network;
+        })
+        ?.filter((network) => network !== 'development'),
+    );
+
+    return Array.from(result ?? []) as Network[];
+  };
+
+  const getAccounts = (): string[] => {
```

Comment:
See comment above

Reply:
same for accounts, this is only for separate accounts of a network visually